.github/ISSUE_TEMPLATE/airflow_bug_report.yml

```diff
@@ -31,17 +31,26 @@ body:
         - "Other Airflow 2 version (please specify below)"
     validations:
       required: true
+  - type: input
+    attributes:
+      label: If "Other Airflow 2 version" selected, which one?
```

Comment:
If not mistaken we delebertly removed all previous versions from the report.
We need to look up the Pr we discussed it.

If the user answe is Other and he does not make a compeling case why issue over older version needs to  be invistigate then the default answer is - please reproduce in main/latest branch and if reproduceable then we can handle it.

My goal eventually is to phase out the old labels for 2.0, 2.1, 2.3 and even 2.4

Reply:
Absolutely. My thinking was to try and short-circuit that response for the triage team and maintainers and add even more encouragement, in a second note, to please reproduce on a recent Airflow version if the author knows it's an older one.

I'm thinking about these scenarios:
1. Nowhere in the details is an Airflow version specified (either being well- or sparsely-documented).
   - Someone trying to triage the issue has to ask the author what version they were on and await a response. The version they are using could be the previous patch version or a few minor versions old; we don't know. The former makes it far more reasonable to address than the latter.

2. The Airflow version used is buried within _any_ of the other fields.
   - Now someone needs to read most of the issue to find out they are on a version that is a year old.

In both cases, it would be nice to know, upfront, what the version is to either continue reading and asses how compelling the issue is, or immediately ask the author to attempt to reproduce on a recent version. Since this new field is at the beginning of the form, If this field isn't specified when "Other Airflow 2 version" is selected, then the triage approach could be to immediately ask to reproduce as well.

Maybe I'm wrong about adding speed for triaging, but more folks continue to use Airflow (which is fantastic) and more non-trivial functionality is getting added (again, fantastic) which means more issues will be logged. Putting a little bit more onus on the issue author and providing a little more context for a "triager" to act upon.

Reply:
Yeah - I think there were two motivations to remove the versions from list:

1) It became far too long to be useful in terms of UX
2) we wanted to make people a bit aware that something they are reporting on is not "current" 

I think adding separate input field does not vioiate any of this.

I am even tempted to use it to strengthen 2) a bit. . Why don't we add a comment / Hint to that field similar to:

`The current versions of Airlfow can selected from the dro-down above and you are advised to upgrade and check if your issue is solved there. If you use old version of Airflow and you cannot upgrade, type your version here - but not we might advise you to ugprade first".


Maybe not as aggressive but i think something like that could help a bit with communicating what we want to communicate.

Reply:
Yeah. It's cool :)